src/algebra/group_ring_action.lean

```diff
@@ -73,6 +83,10 @@ def mul_semiring_action.to_semiring_hom [mul_semiring_action M R] (x : M) : R 
   map_mul' := smul_mul' x,
   .. distrib_mul_action.to_add_monoid_hom M R x }
 
+theorem injective_to_semiring_hom [faithful_mul_semiring_action M R] :
```

Comment:
Do we have a way to go back from an (injective) semiring hom to a (faithful) mul_semiring_action?

Reply:
What is the point?

Reply:
I think it would be a useful part of the API... If the backward direction is already there for mul_semiring_actions, it would be trivial to add it as well for faithful ones. Otherwise, it should probably wait for a future PR.